src/hooks/useChatContext/useChatContext.ts

```diff
@@ -0,0 +1,10 @@
+import { useContext } from 'react';
+import { ChatContext } from '../../components/ChatProvider';
+
+export default function useVideoContext() {
```

Comment:
Should be `useChatContext`.

```suggestion
export default function useChatContext() {
```